### PR TITLE
T3T1 related tooling chores

### PR DESCRIPTION
**Pull request by mmilata**
- build with production bootloader in CI
- T3T1 changelog in `generate-changelogs.py`

### CODE REVIEW

File: tools/towncrier.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@ directory = ".changelog.d"
 filename = "CHANGELOG.md"
 template = ".towncrier.template.md"
 start_string = "The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)."
-title_format = "\n## {version} [{project_date}]"
+title_format = "\n## [{version}] ({project_date})"
```
Comment (matejcik) [inline tools/towncrier.toml:6]: sure about this? last time I touched the global towncrier.toml I cried.
there are local versions in the subdirs

Comment (mmilata) [reply tools/towncrier.toml:6]: i tried generating `core/CHANGELOG.md` and it looked alright, will try the others

**Review by matejcik — APPROVED** [review]

File: tools/towncrier.toml (hunk 1 above)
Comment (matejcik) [reply tools/towncrier.toml:6]: just checking. otherwise utACK, go ahead with the PR if this is indeed correct

Comment (mmilata) [reply tools/towncrier.toml]: you're right, this breaks the format of the other monorepo subdirectories
f61f0a2659110835b758d69d8d7b6484a68e95a9

**Review by matejcik — APPROVED** [review]

**Pull request merged**